integration/findandmodify_compat_test.go

```diff
@@ -190,19 +190,21 @@ func TestFindAndModifyCompatUpdate(t *testing.T) {
 			},
 			skipForTigris: "schema validation would fail",
 		},
-		"InvalidOperator": {
```

Comment:
Why were these tests removed? If they were duplicate of other tests, it makes sense.